content/blog/2024-05-23-roadmap-may-2024.md

```diff
@@ -0,0 +1,250 @@
++++
+title = "May-July roadmap for Linebender"
+authors = ["Olivier Faure"]
++++
+
+On May 7 and 8, 2024, folks from the Linebender team went to [the RustNL conference](https://2024.rustnl.org/) for two days, followed by two days of Unconference.
+
+By that point, we all felt fairly exhausted, but Raph Levien being the cruel taskmaster that he is, he still made us work one extra day, and so we came together in our hotel's conference room to discuss the Linebender project, and what we wanted from it.
```

Comment:
I understand that this is a joke. However a surprising number of people failed to understand that [Raph's April 1st C++ post](https://raphlinus.github.io/rust/2023/04/01/rust-to-cpp.html) was a joke and that was a much more obvious joke. For that reason I recommend against using our blog for such banter, as too many people will take it literally.

```suggestion
By that point, we all felt fairly exhausted, but we still came together in our hotel's conference room to discuss the Linebender project, and what we wanted from it.
```

Reply:
I'm conflicted on this; I think it's probably right to remove it, but I wish it wasn't :laughing:

Reply:
Yeah, that joke was funnier when I wrote it right after hell week. ^^

I'll soften it a bit, and make the joke framing more obvious.

Reply:
(Also, that joke is a lot funnier when you know Raph and how laid back he is, but people reading a blog post probably won't know that, so the framing should be obvious.)